gulpfile.js

```diff
@@ -204,16 +221,38 @@ gulp.task('build', ['clean', 'jshint'], function (callback) {
 	], [
 		'concat-app-amd',
 		'concat-app-commonjs'
+	], [
+		'apply-versions'
 	], callback);
 });
 
 gulp.task('dist', ['build'], function () {
-	return gulp.src([
-		'public/js/ractivef-amd.js',
-		'public/js/ractivef-base.js',
-		'public/js/ractivef-cjs.js',
-		'public/manifest-rf.json'
-	]).pipe(gulp.dest('dist'));
+
+	return mergeStream(
+		gulp.src([
+			'./public/js/ractivef-*.js',
+			'./public/manifest-rf.json',
+			'./public/js/lodash-compat/*',
+			'./public/js/hammerjs/hammer.min.js',
+			'./public/js/ractive-touch/*'
+		], { base: process.cwd() })
+		.pipe(plugins.rename(function (path) {
+			// If file is the default npm name 'index', rename to folder name.
```

Comment:
What do you think about defining the function outside the pipe and just invoking it here @nathf ?